From: Tobias Quathamer <toddy@debian.org>
Date: Wed, 2 Mar 2011 06:00:59 +0100
Subject: Add -lz and -lpthread to linker flags

When using binutils-gold, the linker uses --no-add-needed by default,
resulting in an FTBFS.

Closes: #554082
Thanks: Ilya Barygin <barygin@gmail.com>
---
 src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index f2fc82f..209fd62 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -629,7 +629,7 @@ ENDIF(USE_PYTHON)
 # command. Never remove that line :-)
 # Add ${QT_QTTEST_LIBRARY} below to add the Qt Test library as well
 # Add ${POPPLERQT4_LIBRARY} ${POPPLER_LIBRARY} to reactivate poppler libraries
-TARGET_LINK_LIBRARIES(canorus ${QT_LIBRARIES} ${RUBY_LIBRARY} ${PYTHON_LIBRARY} )
+TARGET_LINK_LIBRARIES(canorus ${QT_LIBRARIES} ${RUBY_LIBRARY} ${PYTHON_LIBRARY} -lz -lpthread )
 # Duma leads to a crash on libfontconfig with Ubuntu (10.04/12.04)
 # duma )
 
